fusion-plugin-service-worker/fixture-apps/app/src/main.js

```diff
@@ -16,6 +16,16 @@ import MockErrorPlugin from './plugins/mock-server-error';
 
 import root from './root.js';
 
+const createMockLogger = () => ({
```

Comment:
We should probably just export a mock logger from `fusion-tokens`.  We have quite a few different mock implementations floating around now.

Reply:
yeah, fixture app failed jmporting types import from `fusion-plugin-service-worker/src/__tests__/utils.node.js`